### PR TITLE
Enabled Reactive Neo4j Auditing to generated createdDate on FoodGroup.

### PR DESCRIPTION
**Pull request by AndrewSpeelman**

### CODE REVIEW

File: src/main/java/food/finder/foodfinder/foodgroup/FoodGroup.java [hunk 1]
```diff
@@ -3,11 +3,8 @@
 import food.finder.foodfinder.user.User;
 import lombok.Getter;
 import lombok.Setter;
-import org.springframework.data.neo4j.core.schema.GeneratedValue;
-import org.springframework.data.neo4j.core.schema.Id;
-import org.springframework.data.neo4j.core.schema.Node;
-import org.springframework.data.neo4j.core.schema.Property;
-import org.springframework.data.neo4j.core.schema.Relationship;
+import org.springframework.data.annotation.CreatedDate;
+import org.springframework.data.neo4j.core.schema.*;
```
Comment (AndrewSpeelman) [inline src/main/java/food/finder/foodfinder/foodgroup/FoodGroup.java]: And my wild card import seems lower than yours apparently

Comment (kcnklub) [reply src/main/java/food/finder/foodfinder/foodgroup/FoodGroup.java]: I would say remove this wildcard import. This is one standard that I have picked up from the IVC world

**Review by kcnklub — CHANGES_REQUESTED** [review]

File: src/main/java/food/finder/foodfinder/foodgroup/FoodGroup.java (hunk 1 above)
Comment (AndrewSpeelman) [reply src/main/java/food/finder/foodfinder/foodgroup/FoodGroup.java]: Man I can't figure out how to get it to NOT do that XD

Comment (AndrewSpeelman) [reply src/main/java/food/finder/foodfinder/foodgroup/FoodGroup.java]: Got it!

**Review by kcnklub — APPROVED** [review]

**Pull request merged**